src/bundle/views/BundleOverview.tsx

```diff
@@ -0,0 +1,316 @@
+import { useMutation } from '@apollo/react-hooks';
```

Comment:
Deze view is bijna 1-op-1 hetzelfde als de CollectionOverview, mss generieker maken adhv het type_id dat meegegeven moet worden aan de query?

Reply:
idd, fixed in: https://github.com/viaacode/avo2-client/commit/2b6a894c3e111b814ab4e74eb942346a87f72e1d